capstone-backend/app.js

```diff
@@ -159,7 +254,7 @@ app.post ('/profile/interests', async (req, res) => {
         query.equalTo("User1", currentUser);
         query.equalTo("name", uiSkills.name);
         const results = await query.find();
-        if ( !results[0] ){
+        if (!results[0]){
```

Comment:
This can be expressed better, 
Are we trying to find out if the first element is null or undefined? `if (results[0] == null)`
Are we trying to find out if results set is empty? `if (results.length === 0)`